components/Button.js

```diff
@@ -23,7 +24,14 @@ const Button = ({
   margin = 0,
   title
 }) => (
-  <button id={id} onClick={onClick} className={className} disabled={disabled} style={style}>
+  <button
+    id={id}
+    onClick={onClick}
+    className={className}
+    disabled={disabled}
+    style={style}
+    data-cy={cypressElementId}
```

Comment:
Is it possible to just pass `Button` `data-cy` directly instead of renaming the prop `cypressElementId`? 🙂

Reply:
Sure I can change that.

Reply:
done :)